From 6346f0a53466176a4d6cb3b72721518a4c5890a0 Mon Sep 17 00:00:00 2001
From: Willy Tarreau <w@1wt.eu>
Date: Sat, 10 May 2014 11:04:39 +0200
Subject: [PATCH 69/70] DOC: stop referencing the slow git repository in the
 README

git.1wt.eu is painfully slow and some people experience issues with
it. Better hide it and only advertise git.haproxy.org which is mirrored
on a faster server.

Also replace haproxy.1wt.eu with www.haproxy.org in the download URL
which appears in the stats page.
---
 README                   | 10 +++-------
 include/common/version.h |  6 +++---
 2 files changed, 6 insertions(+), 10 deletions(-)

diff --git a/README b/README
index a5085fd..4f39df8 100644
--- a/README
+++ b/README
@@ -338,7 +338,7 @@ The list of past and regular contributors is available below. It lists not only
 significant code contributions (features, fixes), but also time or money
 donations :
 
-                        http://haproxy.1wt.eu/contrib.html
+                        http://www.haproxy.org/contrib.html
 
 Note to contributors: it's very handy when patches comes with a properly
 formated subject. There are 3 criteria of particular importance in any patch :
@@ -526,12 +526,8 @@ want, without making your mistakes visible to the rest of the world. It will
 definitely help you contribute quality code and take other people's feedback
 in consideration. In order to clone the HAProxy Git repository :
 
-    $ git clone http://git.1wt.eu/git/haproxy-1.4.git    (stable 1.4)
-    $ git clone http://git.1wt.eu/git/haproxy.git/       (development)
-
-The site above is slow, a faster mirror is maintained up to date here :
-
-    $ git clone http://master.formilux.org/git/people/willy/haproxy.git/
+    $ git clone http://git.haproxy.org/git/haproxy-1.4.git    (stable 1.4)
+    $ git clone http://git.haproxy.org/git/haproxy.git/       (development)
 
 If you decide to use Git for your developments, then your commit messages will
 have the subject line in the format described above, then the whole description
diff --git a/include/common/version.h b/include/common/version.h
index ce6e600..8f1cf37 100644
--- a/include/common/version.h
+++ b/include/common/version.h
@@ -39,19 +39,19 @@
 #ifdef  CONFIG_PRODUCT_URL
 #define PRODUCT_URL    CONFIG_PRODUCT_URL
 #else
-#define PRODUCT_URL    "http://haproxy.1wt.eu/"
+#define PRODUCT_URL    "http://www.haproxy.org/"
 #endif
 
 #ifdef  CONFIG_PRODUCT_URL_UPD
 #define PRODUCT_URL_UPD  CONFIG_PRODUCT_URL_UPD
 #else
-#define PRODUCT_URL_UPD "http://haproxy.1wt.eu/#down"
+#define PRODUCT_URL_UPD "http://www.haproxy.org/#down"
 #endif
 
 #ifdef  CONFIG_PRODUCT_URL_DOC
 #define PRODUCT_URL_DOC  CONFIG_PRODUCT_URL_DOC
 #else
-#define PRODUCT_URL_DOC "http://haproxy.1wt.eu/#docs"
+#define PRODUCT_URL_DOC "http://www.haproxy.org/#docs"
 #endif
 
 #ifdef CONFIG_HAPROXY_VERSION
-- 
1.8.3.2

